Backends/PostScript/font/standard-metrics.lisp

```diff
@@ -1,5 +1,6 @@
 
-(in-package :clim-postscript) 
+(in-package :clim-postscript-font)
+
```

Comment:
Yeah, it's used by both the postscript and PDF backends. The reason it's not clim-document-font, is that I thought we were dealing with postscript fonts, but I guess you're right. Could be renamed.